modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-parameters_{context}"]
+= Ingress controller configuration parameters
+
+The `ingress.config.openshift.io` resource offers the following
```

Comment:
Need to verify.

Reply:
should be `ingresscontrollers.operator.openshift.io` resource.

Reply:
Thank you! Replaced.